### PR TITLE
add PostInitPolarisSDKContextTest

### PR DESCRIPTION
**Pull request by weihubeats**
## add PostInitPolarisSDKContextTest

Feature.

## Describe what this PR does for and how you did.
add PostInitPolarisSDKContextTest

## Adding the issue link (#xxx) if possible.
https://github.com/Tencent/spring-cloud-tencent/issues/240

## Checklist

- [ ] Add copyright holder at the beginning of .class file if it is new.
- [ ] Add information of this PR to CHANGELOG.md in root of project.
- [ ] All junit tests passing.
- [ ] Coverage from `Codecov Report` should not decrease.

### CODE REVIEW

**Comment by weihubeats** [comment]
I'm so sorry I accidentally deleted my fork repository

File: spring-cloud-tencent-polaris-context/src/test/java/com/tencent/cloud/polaris/context/PostInitPolarisSDKContextTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		new PostInitPolarisSDKContext(sdkContext, staticMetadataManager);
+		String regionName = valueContext.getValue(LocationLevel.region.name());
+		String zoneName = valueContext.getValue(LocationLevel.zone.name());
+		String campusName = valueContext.getValue(LocationLevel.campus.name());
+
+		Assertions.assertThat(regionName).isEqualTo(REGION);
+		Assertions.assertThat(zoneName).isEqualTo(ZONE);
+		Assertions.assertThat(campusName).isEqualTo(CAMPUS);
+	}
+}
```
Comment (SkyeBeFreeman) [inline spring-cloud-tencent-polaris-context/src/test/java/com/tencent/cloud/polaris/context/PostInitPolarisSDKContextTest.java:69]: 文件尾添加空行

**Review by SkyeBeFreeman — CHANGES_REQUESTED** [review]
Add info of PR to CHANGELOG.md in the root of project.

**Comment by SkyeBeFreeman** [comment]
> I'm so sorry I accidentally deleted my fork repository

Never mind😆

**Comment by weihubeats** [comment]
done

**Review by SkyeBeFreeman — APPROVED** [review]
LGTM.

**Pull request merged**